src/test/kotlin/com/hedvig/underwriter/web/QuoteBuilderControllerTest.kt

```diff
@@ -43,7 +43,7 @@ internal class QuoteBuilderControllerTest {
         {
             "dateStartedRecievingQuoteInfo": "2019-09-17T13:32:00.783981Z",
             "apartmentProductSubType": "RENT",
-            "incompleteQuoteData": {
+            "IncompleteQuoteData": {
```

Comment:
```suggestion
            "incompleteQuoteData": {
```

Should this not be like this?